libraries/botbuilder-lg/src/templates.ts

```diff
@@ -6,39 +6,39 @@
  * Licensed under the MIT License.
  */
 
-import { LGTemplate } from './lgTemplate';
-import { LGImport } from './lgImport';
+import { Template } from './template';
+import { TemplateImport } from './templateImport';
 import { Diagnostic, DiagnosticSeverity } from './diagnostic';
 import { ExpressionParser } from 'adaptive-expressions';
-import { ImportResolverDelegate } from './lgParser';
+import { ImportResolverDelegate } from './templateParser';
 import { Evaluator } from './evaluator';
 import { Expander } from './expander';
 import { Analyzer } from './analyzer';
-import { LGParser } from './lgParser';
+import { TemplateParser } from './templateParser';
 import { AnalyzerResult } from './analyzerResult';
-import { LGErrors } from './lgErrors';
-import { LGExtensions } from './lgExtensions';
+import { TemplateErrors } from './templateErrors';
+import { TemplateExtensions } from './templateExtensions';
 
 /// <summary>
 /// LG entrance, including properties that LG file has, and evaluate functions.
 /// </summary>
-export class LGFile {
+export class Templates {
 
     /// <summary>
     /// Gets or sets templates that this LG file contains directly.
     /// </summary>
     /// <value>
     /// templates that this LG file contains directly.
     /// </value>
-    public templates: LGTemplate[];
+    public items: Template[];
```

Comment:
i know it's not typical to extends array<T> in JS/TS, so it LGTM to call this items. @tomlm any ideas.

Reply:
yeah the problem was we couldn't have property name same as class name in C#

---
In reply to: [390280511](https://github.com/microsoft/botbuilder-js/pull/1883#discussion_r390280511) [](ancestors = 390280511)